test/test_go.rb

```diff
@@ -254,164 +245,172 @@ def test_vanilla
-    tmux.until { |lines| lines[-2].include?('(6)') }
+    tmux.until { |lines| assert_equal '  10/10 (6)', lines[-2] }
     tmux.send_keys 'C-M'
     assert_equal %w[3 2 5 6 8 7], readonce.split($INPUT_RECORD_SEPARATOR)
   end
 
   def test_multi_max
     tmux.send_keys "seq 1 10 | #{FZF} -m 3 --bind A:select-all,T:toggle-all --preview 'echo [{+}]/{}'", :Enter
 
-    tmux.until { |lines| lines.item_count == 10 }
+    tmux.until { |lines| assert_equal 10, lines.item_count }
 
     tmux.send_keys '1'
     tmux.until do |lines|
-      lines[1].include?('[1]/1') && lines[-2].include?('2/10')
+      assert_includes lines[1], ' [1]/1 '
+      assert lines[-2]&.start_with?('  2/10 ')
     end
 
     tmux.send_keys 'A'
     tmux.until do |lines|
-      lines[1].include?('[1 10]/1') && lines[-2].include?('2/10 (2/3)')
+      assert_includes lines[1], ' [1 10]/1 '
+      assert lines[-2]&.start_with?('  2/10 (2/3) ')
     end
 
     tmux.send_keys :BSpace
-    tmux.until { |lines| lines[-2].include?('10/10 (2/3)') }
+    tmux.until { |lines| assert lines[-2]&.start_with?('  10/10 (2/3) ') }
 
     tmux.send_keys 'T'
     tmux.until do |lines|
-      lines[1].include?('[2 3 4]/1') && lines[-2].include?('10/10 (3/3)')
+      assert_includes lines[1], ' [2 3 4]/1 '
+      assert lines[-2]&.start_with?('  10/10 (3/3) ')
     end
 
     %w[T A].each do |key|
       tmux.send_keys key
       tmux.until do |lines|
-        lines[1].include?('[1 5 6]/1') && lines[-2].include?('10/10 (3/3)')
+        assert_includes lines[1], ' [1 5 6]/1 '
+        assert lines[-2]&.start_with?('  10/10 (3/3) ')
       end
     end
 
     tmux.send_keys :BTab
     tmux.until do |lines|
-      lines[1].include?('[5 6]/2') && lines[-2].include?('10/10 (2/3)')
+      assert_includes lines[1], ' [5 6]/2 '
+      assert lines[-2]&.start_with?('  10/10 (2/3) ')
     end
 
     [:BTab, :BTab, 'A'].each do |key|
       tmux.send_keys key
       tmux.until do |lines|
-        lines[1].include?('[5 6 2]/3') && lines[-2].include?('10/10 (3/3)')
+        assert_includes lines[1], ' [5 6 2]/3 '
+        assert lines[-2]&.start_with?('  10/10 (3/3) ')
       end
     end
 
     tmux.send_keys '2'
-    tmux.until { |lines| lines[-2].include?('1/10 (3/3)') }
+    tmux.until { |lines| assert lines[-2]&.start_with?('  1/10 (3/3) ') }
 
     tmux.send_keys 'T'
     tmux.until do |lines|
-      lines[1].include?('[5 6]/2') && lines[-2].include?('1/10 (2/3)')
+      assert_includes lines[1], ' [5 6]/2 '
+      assert lines[-2]&.start_with?('  1/10 (2/3) ')
     end
 
     tmux.send_keys :BSpace
-    tmux.until { |lines| lines[-2].include?('10/10 (2/3)') }
+    tmux.until { |lines| assert lines[-2]&.start_with?('  10/10 (2/3) ') }
 
     tmux.send_keys 'A'
     tmux.until do |lines|
-      lines[1].include?('[5 6 1]/1') && lines[-2].include?('10/10 (3/3)')
+      assert_includes lines[1], ' [5 6 1]/1 '
+      assert lines[-2]&.start_with?('  10/10 (3/3) ')
```

Comment:
Any reason you put an extra space at the end? e.g. `'  10/10 (3/3) '`

Reply:
For consistency with e.g. `'  2/10 '`, which should match `2/10` but not `2/100`, which, granted, won't actually happen in this case, but seems technically more correct?

Reply:
The other strings that end with `)` don't need extra space.

Reply:
:heavy_check_mark: Removed.